### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by jonei005**
The dashboard page should now have:

* Ability to add new boards.
  * Only accepts names with letters, numbers, spaces, dashes and underscores. No other special characters.
* A new tool menu when hovering over each board tile, where you will _eventually_ be able to rename, share, unlink, and delete the board. Only delete currently works
* Delete board with tool menu
  * Bug: It finds the first board of that name and deletes it. My next plan is to switch to creating a unique ID for each board, and we can delete based on that.

Note: Code is now fully commented.

### CODE REVIEW

File: my-app/src/App.css [hunk 1]
```diff
@@ -45,3 +55,47 @@ div#new-board-form {
...
+  text-transform: none;
+  padding-top: 34px;
+}
+
+#personal-boards .dashboard-board-tools {
+  position: relative;
+  width: 100%;
+  height: 40px;
+  border-radius: .3rem;
+  margin-bottom: -40px;
```
Comment (lepatrick714) [inline my-app/src/App.css]: Alphabetical order

File: my-app/src/App.css [hunk 2]
```diff
@@ -45,3 +55,47 @@ div#new-board-form {
...
+#personal-boards .dashboard-board-tools .tool-btn {
+  position: relative;
+  height: 100%;
+  width: 21%;
+  font-size: 1.2rem;
+  border: 0;
+  background: 0;
+  margin: 0 2%;
+  color: #333;
+}
```
Comment (lepatrick714) [inline my-app/src/App.css]: Here too - Alphabetical order

**Review by lepatrick714 — DISMISSED** [review]

**Review by RBoshae — APPROVED** [review]
Great Comments. Golden UI. Nice work!

File: my-app/src/App.css (hunk 2 above)
Comment (jonei005) [reply my-app/src/App.css]: Do you mean put the class names in alphabetical order, or each attribute? Like background, then color, then margin?

**Review by lepatrick714 — APPROVED** [review]

**Pull request merged**